satellite/console/consoleweb/server.go

```diff
@@ -105,13 +109,18 @@ func (s *Server) grapqlHandler(w http.ResponseWriter, req *http.Request) {
 		ctx = console.WithAuth(ctx, auth)
 	}
 
+	rootObject := make(map[string]interface{})
+	//TODO: add public address to config for production
+	rootObject["origin"] = "http://" + s.listener.Addr().String() + "/"
```

Comment:
We should make https:// the default across the board in my opinion

Reply:
Yeah, that makes sense, but with publicdomain address, this is temporary